src/main/java/com/flightstats/hub/metrics/PeriodicMetricEmitter.java

```diff
@@ -26,7 +25,6 @@ protected void runOneIteration() {
             metricsService.gauge("s3.writeQueue.threads", HubProperties.getS3WriteQueueThreads());
             metricsService.gauge("spoke.write.ttl", HubProperties.getSpokeTtlMinutes(SpokeStore.WRITE));
             metricsService.gauge("spoke.read.ttl", HubProperties.getSpokeTtlMinutes(SpokeStore.READ));
-            metricsService.count(S3Verifier.MISSING_ITEM_METRIC_NAME, 0);
```

Comment:
I was going to add the other metrics from the S3Verifier here, but Paul mentioned that statsd counts/increments reset on flush.